frontend/src/pages/DetailedCourse/DetailedCourse.tsx

```diff
@@ -63,101 +63,114 @@ const DetailedCourse: React.FC<IProps> = ({
   isProgressBarDisplayed,
   isAdmin,
   isCourseCompleted,
-}) => (
-  <AuthorizedLayout pageName="Course">
-    <DetailedCourseWrapper>
-      <BackLink to={page === PAGES.coursesList ? PATHS.coursesList : PATHS.myCourses}>
-        <BackButton variant="medium" color="primary">
-          {ButtonLabels.back}
-        </BackButton>
-        <BackArrow alt="" src={backIconMobile} />
-      </BackLink>
-      <MobileSearchWrapper>
-        <MobileSearch />
-      </MobileSearchWrapper>
-      <InnerWrapper>
-        <ImageWrapper imageUrl={commonCourseData.avatar} />
-        {isProgressBarDisplayed && (
-          <ProgressBar
-            size="large"
-            text={progressText}
-            textColor={PROGRESS_COLOR}
-            variant={progressVariant}
-            value={progressValue}
-          />
-        )}
-        <DetailedCourseTitle>{commonCourseData.title}</DetailedCourseTitle>
-        {isFullTextOpen ? (
-          <DetailedCourseTextMobile>{commonCourseData.description}</DetailedCourseTextMobile>
-        ) : (
-          <DetailedCourseTextMobile>
-            {commonCourseData.description.slice(0, 140)}
-            <ButtonFullText onClick={toggleFullText}>
-              {!isFullTextOpen && OPEN_FULL_TEXT}
-            </ButtonFullText>
-          </DetailedCourseTextMobile>
-        )}
-        <DetailedCourseText>{commonCourseData.description}</DetailedCourseText>
-        <DetailedCourseActionsBox>
-          <CourseInfoBox>
-            <CourseInfo
-              duration={convertDurationToString(commonCourseData.duration)}
-              lessons={commonCourseData.lessons}
-              type={INFO.detailedCourse}
+}) => {
+  const {
+    title: courseTitle,
+    description: courseDescription,
+    duration: courseDuration,
+    lessons: courseLesson,
+    avatar: courseAvatar,
+  } = commonCourseData;
+  const exceededNum = 140;
```

Comment:
please rename to uppercase (EXCEEDED_NUM) and put it at the beginning of the file